proto/ca_service.proto

```diff
@@ -40,17 +40,8 @@ service IstioCAService {
 message Request {
   // PEM-encoded certificate signing request
   bytes csr_pem = 1;
-  NodeAgentCredentials node_agent_credentials = 2;
-}
-
-message NodeAgentCredentials {
-  oneof node_credential {
-    GcpInstanceCredential gcp_instance_credential = 1;
-  }
-}
-
-message GcpInstanceCredential {
-  bytes signed_instance_metadata = 1;
+  // opaque credential for node agent
```

Comment:
what do you mean by opaque?

Reply:
maybe it's not a good way to use the word... but I mean it's just a byte array and it's up to the authenticator to interpret it.